### PR TITLE
i.sentinel.parallel.download: allow download from GCS with scenename specifications and without credentials

### PR DESCRIPTION
**Pull request by griembauer**
This PR allows `i.sentinel.parallel.download` to directly download from GCS if the sentinel-2 scene names are specified via the `-s `flag and if **no settings file is provided**. This specific case is already possible in `i.sentinel.download` but so far the `settings` parameter was required in `i.sentinel.parallel.download`

### CODE REVIEW

File: src/imagery/i.sentinel/i.sentinel.parallel.download/i.sentinel.parallel.download.py [hunk 1]
```diff
@@ -164,10 +165,13 @@ def scenename_split(scenename, datasource, esa_name_for_usgs=False):
         end_day_dt = dt_obj + timedelta(days=1)
         start_day = start_day_dt.strftime("%Y-%m-%d")
         end_day = end_day_dt.strftime("%Y-%m-%d")
-        # get query string
-        if not scenename.endswith(".SAFE"):
-            scenename = scenename + ".SAFE"
-        query_string = "filename={}".format(scenename)
+        if datasource == "GCS":
+            query_string = f"identifier={scenename.replace('.SAFE', '')}"
```
Comment (metzm) [inline src/imagery/i.sentinel/i.sentinel.parallel.download/i.sentinel.parallel.download.py:169]:
```suggestion
            query_string = f"identifier={scenename.removesuffix('.SAFE')}"
```
Here you remove the suffix `.SAFE` ...

Comment (griembauer) [reply src/imagery/i.sentinel/i.sentinel.parallel.download/i.sentinel.parallel.download.py:169]: `removesuffix` was introduced in Python 3.9 - not sure if we can expect this?

File: src/imagery/i.sentinel/i.sentinel.parallel.download/i.sentinel.parallel.download.py [hunk 2]
```diff
@@ -164,10 +165,13 @@ def scenename_split(scenename, datasource, esa_name_for_usgs=False):
...
+        else:
+            # get query string
+            if not scenename.endswith(".SAFE"):
+                scenename = scenename + ".SAFE"
```
[the remark this answers is not in the document]
Comment (metzm) [reply src/imagery/i.sentinel/i.sentinel.parallel.download/i.sentinel.parallel.download.py:173]: Ah ok, then I misunderstood the manual:
"Querying and filtering data on Google Cloud Storage requires the fee-based usage of [BigQuery](https://cloud.google.com/bigquery/external-data-cloud-storage?hl=en), which is currently not supported by i.sentinel.download. Instead, when using the datasource=GCS option, querying is performed using the [Copernicus Open Access Hub](https://scihub.copernicus.eu/) interface, while only the download step makes use of Google Cloud Storage."

File: src/imagery/i.sentinel/i.sentinel.parallel.download/i.sentinel.parallel.download.py (hunk 1 above)
Comment (metzm) [reply src/imagery/i.sentinel/i.sentinel.parallel.download/i.sentinel.parallel.download.py:169]: Indeed, I missed this. I would not expect/require Python 3.9, forget about the suggestion.

**Review by metzm — APPROVED** [review]
All fine with me, thanks!

**Pull request merged**